### DISCUSSION TITLE
WW 10 - Noam

### DISCUSSION THREAD

**Discussion by NoamRahat** [FPGA-MAFIA/fpga_mafia#533] [Weekly]
# Completed:
- #503 : I have ran sw compilation, hw compilation and evntully ran the hole simulation, and saw the output on the screen.log
- #507 : Thins ;)

# WIP:

# Not Started
- #532 , ETA: Monday evening

**Comment by amichai-bd** [comment]
Thanks for the weekly update!
Can please also add a short comment next to each issue numbers explaining what was done?
Please update existing Post
